validator/lib/validateTileset.js

```diff
@@ -103,29 +104,29 @@ function validateTileHierarchy(root, tilesetDirectory) {
             var tileBox = tile.boundingVolume.box;
 
             if (defined(contentRegion) && defined(tileRegion) && !regionInsideRegion(contentRegion, tileRegion)) {
-                return 'content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']';
+                return errorMessage('content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']', tile.content);
```

Comment:
For all of these just pass in `tile` instead of `tile.content`. The output message needs to show both the tile bounding volume and the content bounding volume.